sentry-samples/sentry-samples-android/src/main/res/drawable/logo_pocket_casts.xml

```diff
@@ -0,0 +1,50 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Should we add an attribution header here? Something along those lines?
```suggestion
<!-- Based on https://github.com/Automattic/pocket-casts-android/blob/main/modules/services/images/src/main/res/drawable/logo_pocket_casts.xml 

This Source Code Form is subject to the terms of the Mozilla Public
License, v. 2.0. If a copy of the MPL was not distributed with this
file, You can obtain one at [http://mozilla.org/MPL/2.0/.](http://mozilla.org/MPL/2.0/)
-->
<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Reply:
ow yes, a very good callout!

Reply:
I actually realized that you cannot use the contributor's logo according to MPL, so I'll just replace it with the sentry one 😅